<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralOverview</class>
 <widget class="QDialog" name="GeneralOverview">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Overview</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>631</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>General</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kingdom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MaxHP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Package</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>620</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="skillLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>30</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Skill</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="skillTextEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playEffecButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>620</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Play effect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lastWordButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Last word</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="skillGroupBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>550</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Skills</string>
   </property>
  </widget>
  <widget class="QLabel" name="generalPhoto">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>221</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>GeneralOverview</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
